src/pages/tBTC/Bridge/DepositDetails.tsx

```diff
@@ -54,35 +57,23 @@ export const DepositDetails: PageComponent = () => {
 
   // Extract deposit details values to use them as a dependency in hook
   // dependency array.
-  const btcDepositTxHash = data?.btcTxHash
-  const amount = data?.amount ?? "0"
-  const confirmations = data?.confirmations
-  const requiredConfirmations = data?.requiredConfirmations
-  const optimisticMintingRequestedTxHash =
-    data?.optimisticMintingRequestedTxHash
-  const optimisticMintingFinalizedTxHash =
-    data?.optimisticMintingFinalizedTxHash
-  const thresholdNetworkFee = data?.treasuryFee
-  const mintingFee = data?.optimisticMintFee
+  const {
+    btcTxHash: btcDepositTxHash,
+    amount,
+    confirmations,
+    requiredConfirmations,
+    depositRevealedTxHash,
+    optimisticMintingRequestedTxHash,
+    optimisticMintingFinalizedTxHash,
+    treasuryFee: thresholdNetworkFee,
+    optimisticMintFee: mintingFee,
+  } = data
 
   useEffect(() => {
-    if (
-      !!btcDepositTxHash &&
-      confirmations !== undefined &&
-      requiredConfirmations !== undefined &&
-      confirmations < requiredConfirmations
-    ) {
-      dispatch(
-        tbtcSlice.actions.fetchUtxoConfirmations({
-          utxo: { transactionHash: btcDepositTxHash, value: amount },
-        })
-      )
-    }
-
```

Comment:
Why are we removing this piece of code here? Without that, the `txConfirmations` in the store won't be updated.

Reply:
Reverted, ref commit: 5d92563d